### ISSUE TITLE
Some subtitles are not displayed at the beginning of the video

### ISSUE THREAD

**Issue by whinette** [plexinc/plex-for-kodi#127]
Hello,

On most of my videos, subtitles aren't displayed at the beginning of the video, but are after ~10 secondes.
If I go back to the beginning of the video, subtitles are displayed.

Logs from a fresh install ; 
[kodi.log](https://github.com/plexinc/plex-for-kodi/files/1335030/kodi.log)

It happens on all my devices.

MediaInfo of the played file in the log :

[mediainfo.txt](https://github.com/plexinc/plex-for-kodi/files/1335043/mediainfo.txt)

Cheers,
Whinette

**Comment by pannal** [comment]
Yeah I've experienced the same issues, particularly tied to embedded subtitles. External ones work flawlessly.
